cmd/gossamer/config.go

```diff
@@ -674,7 +674,7 @@ func setDotNetworkConfig(ctx *cli.Context, tomlCfg ctoml.NetworkConfig, cfg *dot
 	}
 
 	// check --nomdns flag and update node configuration
-	if nomdns := ctx.GlobalBool(NoMDNSFlag.Name); nomdns {
+	if nomdns := ctx.GlobalBool(strings.Split(NoMDNSFlag.Name, ",")[0]); nomdns {
```

Comment:
Why splitting??

Reply:
because of the way urfave/cli V1 handles aliases, this is updated when upgraded to V2.